public/vislib/vector_layer_types/EsLayer.js

```diff
@@ -312,9 +226,11 @@ export default class EsLayer {
     const feature = L.marker(
       toLatLng(hitCoords),
       {
+        className: 'point-popup',
         icon: searchIcon(options.icon, options.color, options.size, overlap),
         pane: 'overlayPane'
       });
+    _.set(feature, 'feature.id', hit._id);
```

Comment:
Does feature.id already exist? As in, could it be possible that someone would like to display that field on a popup? 

Set is nice for very nested stuff, in general the consensus seems to be that if it's simple to do in plain javascript, just use it

```suggestion
    feature.id = hit._id;
```

Reply:
Then it should be ` feature.feature = { id: hit._id };` I think